.prettierignore

```diff
@@ -0,0 +1 @@
+*.md
```

Comment:
Adding a prettier config to this Rust repo is really odd. By this logic, this repo could be filled with config files that are unrelated to this ecosystem 😅 

Lets remove this prettierignore file and add it to gitignore so you can personally use this file without it being committed. Exact same logic as ignoring vscode editor configs 👍

Reply:
Really?
It's a single line hidden file that declares your stated preference regarding the common tool and will save you from reviewing diffs that revert your unique formatting preferences but okay...

Reply:
Yes please